hal/src/rtl872x/hal_platform_rtl8721x_config.h

```diff
@@ -55,7 +55,7 @@
 
 #define HAL_PLATFORM_I2C2 (1)
 
-#define HAL_PLATFORM_USART2 (1)
+#define HAL_PLATFORM_USART2 (0)
```

Comment:
This is the same as on other cellular platforms: we disable `Serial2` in wiring, so that the application does not construct `USARTSerial` object for it causing `hal_usart_init()` call which messes things up, as the initialization happens in NCP client.

Reply:
But on P2 user can use `Serial2` right? It's Wi-Fi only.

Reply:
Ah, that's true :+1: This needs to be in platform-specific header, good catch.